specification/appendix.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Appendix
```

Comment:
Let's give this a name, like "Appendix A: Included Utilities"

Reply:
Updated to `Appendix-A`